ArmPlatformPkg/Drivers/PL061GpioDxe/PL061Gpio.h

```diff
@@ -37,6 +37,6 @@
 #define PL061_GPIO_PINS  8
 
 // All bits low except one bit high, native bit length
-#define GPIO_PIN_MASK(Pin)  (1UL << ((UINTN)(Pin)))
+#define GPIO_PIN_MASK(Pin)  (LShiftU64(1UL, Pin))

```

Comment:
The Pin value should be only 8-bits not UINTN in length and hence shouldn't be shifted to a larger value. It might be better to make sure we aren't shifting more than 7 bits and we can use something like this

`#define GPIO_PIN_MASK(Pin) (UINT8)(1 << (Pin & (PL061_GPIO_PINS - 1)))`

Reply:
done